base/ca/shared/profiles/ca/caCMCECserverCertWithCRLDP.cfg

```diff
@@ -106,8 +102,6 @@ policyset.serverCertSet.10.default.class_id=crlDistributionPointsExtDefaultImpl
 policyset.serverCertSet.10.default.name=CRL Distribution Points Extension Default
 policyset.serverCertSet.10.default.params.crlDistPointsCritical=false
 policyset.serverCertSet.10.default.params.crlDistPointsEnable_0=true
-policyset.serverCertSet.10.default.params.crlDistPointsIssuerName_0=SET_ME_TO_DN_OF_CRL_ISSUER
-policyset.serverCertSet.10.default.params.crlDistPointsIssuerType_0=DirectoryName
```

Comment:
@fmarco76 I'm unclear on this.  Are we supposed to remove those two or just leave them blank like this? policyset.serverCertSet.9.default.params.crlDistPointsIssuerName_0=
policyset.serverCertSet.9.default.params.crlDistPointsIssuerType_0=